src/nftfeed.sol

```diff
@@ -0,0 +1,148 @@
+// Copyright (C) 2020 Centrifuge
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity >=0.5.15;
+
+import "ds-note/note.sol";
+import "tinlake-auth/auth.sol";
+import "tinlake-math/interest.sol";
+
+contract ShelfLike {
+    function shelf(uint loan) public view returns (address registry, uint tokenId);
+    function nftlookup(bytes32 nftID) public returns (uint loan);
+}
+
+contract PileLike {
+    function setRate(uint loan, uint rate) public;
+    function debt(uint loan) public returns (uint);
+    function pie(uint loan) public returns (uint);
+    function changeRate(uint loan, uint newRate) public;
+    function loanRates(uint loan) public returns (uint);
+    function file(bytes32, uint, uint) public;
+    function rates(uint rate) public view returns(uint, uint, uint ,uint48);
+}
+
+contract BaseNFTFeed is DSNote, Auth, Interest {
```

Comment:
Can't this just be `Math` instead of `Interest` ?

Reply:
I added math